python-sdk/nuimages/nuimages.py

```diff
@@ -694,7 +694,17 @@ def get_segmentation(self,
         nuim_name2idx_mapping = dict()
         # The 0 index is reserved for non-labelled background; thus, the categories should start from index 1.
         for i, c in enumerate(self.category, start=1):
```

Comment:
Sort the categories by name

Reply:
Remove ego vehicle and driveable surface from this enumeration, as otherwise your indices are wrong.